frontend/src/components/panels/project_filters_panel/filter_map_components/CheckBoxRow.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
-import Form from 'react-bootstrap/Form';
 import ProjectTypeMarker from '../../../helpers/projectTypeMarker';
+import styled from 'styled-components'
 
-const CheckBoxRow = ({ Label, ProjectType }) => (
-    <div className="form-check">
-        <input type="checkbox" className="form-check-input"/>
-        <label title={Label} type="checkbox" className="form-check-label">{Label}</label>
-        <ProjectTypeMarker FillClass={ProjectType}/>
-    </div>
+const CheckboxContainer = styled.div`
+  display: inline-block;
+  vertical-align: middle;
+  cursor: pointer;
+`
+
+const Icon = styled.svg`
+  fill: none;
+  stroke: #5B5A5E;
+  stroke-width: 6.5px;
+  width: 12px;
+  padding-bottom: 8px;
+  padding-right: 2px;
+`
+
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  clip: rect(0 0 0 0);
+  clippath: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+`
+
+const StyledCheckbox = styled.div`
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  background: #EEF0EA;
+  border-radius: 2.5px;
+  border: 1px solid darkgray;
+
+  ${Icon} {
+    visibility: ${props => (props.checked ? 'visible' : 'hidden')}
+  }
+`
+
+const CheckBoxRow = ({ Label, ProjectType, checked, toggleView}) => (
+    <CheckboxContainer>
+        <HiddenCheckbox checked={checked} onChange={toggleView} />
+        <StyledCheckbox checked={checked}>
+            <Icon viewBox="0 0 24 24">
+                <polyline points="20 6 9 17 4 12" />
+            </Icon>
+        </StyledCheckbox>
+        <span style={{ marginLeft: 8 }}>{Label}</span>
```

Comment:
Can you assign this css attribute to a class? Thanks

Reply:
@ASTPMeetup I didn't assign a css class b/c I was unsure of where I would put it. Can you give me a hint as to where to put it?

Reply:
frontend/src/styles/panels/_filters_panel.scss.

Each panel has its on css file. If its a global class that needs to be applied to multiple pages it can assign in frontend/src/styles/panels/global.css

Reply:
How to handle style changes has been added to the README document inside `\frontend`